docs/external-files.org

```diff
@@ -1,14 +1,58 @@
+** Reading external files
 There are multiple ways to do it in ZIG:
-1. @embedFile
-Simply going to embed the file in the binary.
-https://ziglang.org/documentation/master/#embedFile
+*** @embedFile
+This method is simply going to embed the file in the binary.
 
-2. Use an allocator to dynamically store the content of the file 
+Beware that by using this method the path to the file is going to be relative to the file you are calling it from and that the file must be in the same package.
+
+By using the other methods below you can use a file from anywhere but relative to your root folder and not main.zig.
```

Comment:
It will embed on compile time ?

Reply:
yes: https://ziglang.org/documentation/master/#embedFile

Reply:
Okay, it's interesting to specify?

Reply:
will do :)